pydarn/plotting/rtp.py

```diff
@@ -841,19 +853,14 @@ def plot_summary(cls, dmap_data: List[dict], beam_num: int = 0,
 
                 # plot time-series parameters that share a plot
                 if i < 2:
-                    with warnings.catch_warnings(record=True) as w:
+                    with warnings.catch_warnings() as w:
+                        warnings.simplefilter("ignore")
```

Comment:
i.e. is this output any different to what it was before?

Reply:
I will add a comment to make things more clearer to the reader. 
The filter is to prevent multiple warnings coming from the other plotting methods. 

The citing warning is all the same. Sadly warnings are not as nice as exceptions so I need to figure out how to encapsulate them.

Reply:
Ah yes, I get it now, sorry, I just didn't read it right the first time.